### ISSUE TITLE
Inf or NaN during training

### ISSUE THREAD

**Issue by Harryqu123** [haonanwang0522/GTPT#5]
Hi there, thanks for your intersting work.

I am opening this issue as I seem to fail to properly train the model following the given code.

Specifically, I tried several times yet every time I trained, the loss will become nan quickly at the first several epochs, as screenshotted below.

Can I kindly know do you have any idea on the possible reason?

My training script is as follows.

python tools/train.py --cfg /home/xxx/GTPT/experiments/coco/gtpt/gtpt_t_256x192_adam_lr1e-3_split2_sigma4.yaml

Many thanks in advance!

![image](https://github.com/user-attachments/assets/93b46ad9-08c9-48a7-bff8-f078a7a2b6c1)
